datastorages/cmis/src/main/java/org/fao/geonet/api/records/attachments/CMISStore.java

```diff
@@ -657,14 +658,22 @@ protected MetadataResourceExternalManagementProperties getMetadataResourceExtern
     ) {
         String metadataResourceExternalManagementPropertiesUrl = cmisConfiguration.getExternalResourceManagementUrl();
         if (!StringUtils.isEmpty(metadataResourceExternalManagementPropertiesUrl)) {
+            // {objectid}  objectId // It will be the type:visibility:metadataId:version:resourceId in base64
```

Comment:
These properties are objects that can be supplied to the third party application.  

As JCLOUD does not have any special ID to identify the record like CMIS did, I added a new {objectId} for JCLOUD so that the 3rd party application could use that value to identify the record.

In order to make some of these parameters consistent between storage solutions, I simply added the new  {objectId} to CMIS as well so that CMIS applications could use it as well if needed.

I can remove it if you think there are issues with adding this new parameters.

Reply:
@ianwallen it's ok to keep it. I just wanted to have clear the reason. Thanks for the clarification.